include/deal.II/base/mpi.h

```diff
@@ -1370,6 +1370,33 @@ namespace Utilities
     std::set<T>
     compute_set_union(const std::set<T> &set, const MPI_Comm &comm);
 
+    /**
+     * Compute exclusive or incluse prefix sum for scalar quantities.
+     */
+    template <typename T>
+    std::pair<T, T>
+    prefix_sum(const T &       value,
+               const MPI_Comm &comm,
+               const bool      exclusive = true);
```

Comment:
```suggestion
    /**
     * Compute exclusive or incluse prefix sum for scalar quantities.
     */
    template <typename T>
    std::pair<T, T>
    prefix_sum(const T &       value,
               const MPI_Comm &comm,
               const bool      exclusive = true);
```
The term "prefix sum" is a computer science term that is not widely understood in the broader community (for example, I have to think about it every time I see it). Would you mind explaining what a "prefix sum" is in the documentation of the function? And what "exclusive" and "inclusive" means? I think a simple formula would do wonders :-)

Separately, why does the function return a *pair* of values? Can you document what the two parts are?